ppx/ppx.ml

```diff
@@ -1285,7 +1259,7 @@ let jsxMapper () =
                                     , Exp.construct {loc; txt= Lident "()"} None
                                     ) ] )]] )
                  in
-                 make_props @@ js_comp @@ outer )
+                 make_props (js_comp outer) )
             in
             let inner_make_ident =
```

Comment:
To keep things working, instead of passing this `inner_make_ident` to `outer_make`, we would pass `js_component_expr`.

Something like:

```ocaml
            let js_component_expr =
              (* I need to revisit how all locations are created... 🙈  *)
              let loc = pstr_loc in
              [%expr
                Js_of_ocaml.Js.Unsafe.js_expr [%e constantString ~loc pval_prim]]
            in
            { pstr_loc
            ; pstr_desc=
                Pstr_value (Nonrecursive, [outer_make js_component_expr])
            }
            :: returnStructures ) )
```